### DISCUSSION TITLE
how do i remove this from my list

### DISCUSSION THREAD

**Discussion by SlyTheFloof** [Rapptz/discord.py#6434] [General]
how do i remove this from my list

**Comment by SlyTheFloof** [comment]
![image](https://user-images.githubusercontent.com/56106431/108286187-a0924a00-7156-11eb-90de-b8db81c9514c.png)

**Reply by Rapptz** [reply]
This isn't the best place for Github support. I'm not sure how to remove items from that list but it's outside of the scope here.
